packages/core/src/components/Table/stories/TableHooks/TableHooks.stories.tsx

```diff
@@ -151,12 +157,6 @@ const UseHvPagination = () => {
     getHvPaginationProps,
   } = useHvData<NewEntry, string>({ columns, data }, useHvPagination);
 
-  const EmptyRow = () => (
-    <HvTableRow>
-      <HvTableCell colSpan={100} />
-    </HvTableRow>
-  );
```

Comment:
Declaring components inside components is definitely an anti-pattern, but moving this outside the sample code makes the sample harder to read...
We should (maybe not now) work on improving this